src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetArea, ToDraw {
```

Comment:
```suggestion
public class Circle extends Figure {
```
it's enough to implement these interfaces in Figure